designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala

```diff
@@ -181,7 +182,9 @@ object QuestDbFEStatisticsRepository extends LazyLogging {
   }
 
   private def createRootDirIfNotExists(): File = {
-    val nuDir = File.temp.createChild("nu", asDirectory = true)
+    val nuDir = File.temp
+      .createChild("nu", asDirectory = true)
+      .createChild(UUID.randomUUID().toString, asDirectory = true)
```

Comment:
So after a restart, we will lose data?

Reply:
Yes, it's workaround to work with multiple instances. If it's not critical we can close PR.

Reply:
Ok, so as a temporary workaround it is fine. Let's only remember to fix it before the release. Maybe some jira task with fix version would help with that?

Reply:
It is not critical IMO, we can leave for a while with this. There is an easy workaround for that - we just need to stop designer during tests and resume it after tests are finished

Reply:
Ok, so I close PR